spec/index.html

```diff
@@ -476,6 +549,16 @@ <h3>Quoted Literals</h3>
         and may be followed by a <a data-cite="RDF12-CONCEPTS#dfn-base-direction">base direction</a>
         separated from the language tag by <a href="#cp-hyphen-hyphen"><code>--</code></a>.</p>
 
+      <p class="note">When using RDF 1.2-specific features, such as
+        <a data-cite="RDF12-CONCEPTS#dfn-base-direction">base direction</a>,
```

Comment:
```suggestion
        <a data-cite="RDF12-CONCEPTS#dfn-base-direction">initial text direction</a>,
```

Reply:
@gkellogg - the link and text in RDF Concept is "base direction" then it says "initial text direction".